app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] != "cp" or len(command) != 3:
+        raise ValueError("Invalid input")
+    file_from = command[1]
+    file_to = command[2]
```

Comment:
Prefer using unpacking instead of indexing in the future to create more descriptive code